### ISSUE TITLE
Foreground colour of selected text is white

### ISSUE THREAD

**Issue by tcak** [serge-rider/dbeaver#876]
It is hard to show this, so I will explain. Basically, when I click into a text field (e.g. Table Name in "Information" tab), the text inside is selected automatically. But the problem is both text is white, and foreground is white as well. So, it is not obvious what is happening. Inside of text field looks completely white. I click one more time, and it is not selected any more. Then text turns black, so I can see it. I changed Ubuntu themes in case it was due that, but nothing has changed.

I checked settings for this, but couldn't have found any colour option for it.

**Comment by serge-rider** [comment]
Not sure I understand the problem.
Could you post a screenshot?

**Comment by tcak** [comment]
I typed into Table Name "FeePercentages", and then selected "nta" part with mouse. Selection text is white, foreground colour is white as well. Picture is below:

![screenshot from 2016-10-23 09-12-44](https://cloud.githubusercontent.com/assets/10595665/19624483/2acda698-9901-11e6-855a-f4df96ea7860.png)

To make sure it is not due to Ubuntu theme (I am using 16.10, Unity Desktop by the way), I changed to between Radiance, Adwaita, Arc. Same for all of them. To be honest, changing the theme is not affecting the those colours at all.
